config.py

```diff
@@ -35,7 +35,9 @@ class Config(object):
         "invite_contributor": "5eefe42d-1694-4388-8908-991cdfba0a71",
         "framework_agreement_signature_page": "08929c93-f9e7-4b40-b75f-418659308324",
         'g-cloud-application-started': '7362aad7-1ffa-4e36-8ddd-d16cc5305175',
-        'digital-outcomes-and-specialists-application-started': '1920c708-66a9-466e-a0ae-3bd13d31e32d'
+        'digital-outcomes-and-specialists-application-started': '1920c708-66a9-466e-a0ae-3bd13d31e32d',
+        'framework-clarification-question': "8574484f-9907-44c0-b9d5-1120beb64ef0",
+        'framework-application-question': '6681d4a1-6e30-407f-9e8b-3f6633d73546'
```

Comment:
```suggestion
        'framework-application-question': '6681d4a1-6e30-407f-9e8b-3f6633d73546',
```

Reply:
I don't understand the addition of the comma here?

Reply:
It makes a nicer diff when you add another element to the dict (see the first line of this diff for instance); it's strictly cosmetic and a matter of personal preference of course.